src/utils/get-core-member-logins.ts

```diff
@@ -13,37 +13,42 @@ limitations under the License.
 
 import * as core from '@actions/core';
 import { CodeOwnersEntry, loadOwners, matchFile } from 'codeowners-utils';
-import { union, uniq } from 'lodash';
+import { uniq, union } from 'lodash';
 import { context } from '@actions/github';
 import { getChangedFilepaths } from './get-changed-filepaths';
 import { map } from 'bluebird';
 import { octokit } from '../octokit';
 
 export const getCoreMemberLogins = async (pull_number: number, teams?: string[]) => {
+  const teamsAndLogins = await getCoreTeamsAndLogins(pull_number, teams);
+  return uniq(teamsAndLogins.map(({ login }) => login));
+};
+
+export const getCoreTeamsAndLogins = async (pull_number: number, teams?: string[]) => {
   const codeOwners = teams ?? (await getCodeOwners(pull_number));
 
   if (!codeOwners?.length) {
-    core.setFailed('No code owners found.');
+    core.setFailed('No code owners found. Please provide a "teams" input or set up a CODEOWNERS file in your repo.');
     throw new Error();
   }
 
-  const adminLogins = await map(codeOwners, async team =>
+  const teamsAndLogins = await map(codeOwners, async team =>
```

Comment:
totally out of curiosity.... does using the bluebird `map` function give us any advantage here over the array method?

Reply:
`bluebird` is specifically meant for asynchronous iteration. That said, I think it's an old library and we could probably replace it 🤷